src/PatchOrchestrationApplication/NodeAgentNTService/src/Manager/WindowsUpdateManager.cs

```diff
@@ -398,7 +478,7 @@ private bool HandleWUOperationStates(CancellationToken cancellationToken)
                             OperationResultCode installResult = InstallUpdates(cancellationToken);
                             reschedule = (installResult != OperationResultCode.orcSucceeded ? true : reschedule);
 
-                            WindowsUpdateOperationResult installationResult = this._operationResultFormatter.FormatInstallationResult(installResult, this._wuCollectionWrapper);
+                            WindowsUpdateOperationResult installationResult = this._operationResultFormatter.FormatInstallationResult(installResult, this._wuCollectionWrapper, this.lastUpdateOperationStartTimeStamp);
```

Comment:
>lastUpdateOperationStartTimeStamp [](start = 181, length = 33)

Where is this being read in this switch case?

Reply:
If installation is in progress, then, it means that whatever the lastUpdateOperationStartTime is saved in the file is correct one.